src/docs/sdk/event-payloads/properties/trace_id.mdx

```diff
@@ -0,0 +1,9 @@
+`trace_id`: 
+
+: _Required_. Determines to which `trace` the Span belonges to.
```

Comment:
(copying from an outdated suggestion from @AbhiPrasad + my previous suggestion to document the content):

```suggestion
: _Required_. Determines which `trace` the Span belongs to. The value should be 16 random bytes encoded as a hex string (32 characters long).
```